hm_pyhelper/miner_param.py

```diff
@@ -50,6 +50,10 @@ def run_gateway_mfr(args):
         LOGGER.exception(err_str)
         raise GatewayMFRFileNotFoundException(err_str) \
             .with_traceback(e.__traceback__)
+    except Exception as e:
```

Comment:
I think it would make more sense to raise an ECCMalfunctionException here instead of swallowing the exception.